docs/extensions/scala.md

```diff
@@ -54,6 +54,9 @@ export interface ScalaTestParams {
   /** The test classes to be run in this test execution.
    * It is the result of `buildTarget/scalaTestClasses`. */
   testClasses?: ScalaTestClassesItem[];
+
+  /** The jvm options to run tests with. */
```

Comment:
Can you elaborate on what happens when this option conflicts with existing JVM options that are configured in the build? Should the two be concatenated or should this override?

Reply:
Done